Jatmika Teja

thejatmik opened the pull request:
CMS Client
https://vue-ecommerce-cms-client.firebaseapp.com/
my user `user01@mail.com:leleyeye`

thejatmik committed:
Init Client

- Init client with vuecli without fetch from server

dhimashary commented on src/router/index.js:
```diff
@@ -0,0 +1,112 @@
+    redirect: {
+      name: 'ProductContainer'
+    },
+    children: [
```
goodjob udah belajar nested routes nya

dhimashary commented on src/components/CreateItem.vue:
```diff
@@ -0,0 +1,33 @@
+</template>
+
+<script>
+export default {
```
component jangan lupa kasih nama + konsisten misal CreateProductForm

thejatmik committed:
Client Data Mockup

- Implement vuex
- Implement user login & register function on client
- mocking fetch, and loading interaction

thejatmik committed:
Implement Client to dev database

- User: Register and login
- Product: fetch and create

thejatmik committed:
Finishing client v1

- CRUD Items
- Login and register
- Error notification
- Interaction to server

thejatmik committed:
Firebase deployment v01

https://vue-ecommerce-cms-client.firebaseapp.com/

dhimashary commented on src/store/index.js:
```diff
@@ -0,0 +1,243 @@
+      state.dbCred.email = payload.email
+      state.dbCred.token = payload.token
+      state._isLogin = true
+      localStorage.setItem(state.appName, JSON.stringify({ token: payload.token }))
```
sebisa mungkin mutation bersih dari hal2 kaya gini ya, jadi mutation benar2 ngubah state doang

dhimashary commented:
next time firebase di ignore ya

thejatmik committed:
Add feature

- Text field for image url in add item page
- Loading page implemented or more feature

thejatmik committed:
refactor

thejatmik committed:
refactor code
